Resources/config/dbal.xml

```diff
@@ -23,10 +22,6 @@
         <service id="Doctrine\DBAL\Driver\Connection" alias="database_connection" public="false" />
         <service id="Doctrine\DBAL\Connection" alias="database_connection" public="false" />
 
-        <service id="doctrine.dbal.logger.chain" class="%doctrine.dbal.logger.chain.class%" public="false" abstract="true" >
```

Comment:
Removing this one is a no-go. It is still needed if some projects add their own sql loggers. the removal must only happen in a new major version, with deprecations before

Reply:
Indeed, thank you

Reply:
I simply reverted this change, from my POV, I can't deprecate this service while `doctrine/dbal` versions without middlewares and `symfony-bridge` without `DebugMiddleware` are still supported. Am I missing something ?

Reply:
indeed. We cannot deprecated it yet